src/components/Dashboard/Native/dashboard.tsx

```diff
@@ -393,6 +437,73 @@ const DashboardNative: React.FunctionComponent = () => {
+                !branches.find((branch) => branch.name == draft.branchName) && (
+                  <GalleryItem key={draft.branchName}>
+                    <Card key={draft.branchName}>
+                      <CardHeader
+                        actions={{
+                          actions: (
+                            <Dropdown
+                              onSelect={() => onActionMenuSelect(draft.branchName)}
+                              toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+                                <MenuToggle
+                                  ref={toggleRef}
+                                  isExpanded={isActionMenuOpen[draft.branchName] || false}
+                                  onClick={() => onActionMenuToggle(draft.branchName, !isActionMenuOpen[draft.branchName])}
+                                  variant="plain"
+                                  aria-label="contribution action menu"
+                                  icon={<EllipsisVIcon aria-hidden="true" />}
+                                />
+                              )}
+                              isOpen={isActionMenuOpen[draft.branchName] || false}
+                              onOpenChange={(isOpen: boolean) => onActionMenuToggle(draft.branchName, isOpen)}
+                              popperProps={{ position: 'end' }}
+                            >
+                              <DropdownList>
+                                <DropdownItem key="edit-contribution" onClick={() => handleEditDraftContribution(draft.branchName)}>
+                                  Edit Contribution
+                                </DropdownItem>
+                                <DropdownItem key="delete-contribution" onClick={() => handleDeleteDraftContribution(draft.branchName)}>
+                                  Delete Contribution
+                                </DropdownItem>
+                              </DropdownList>
+                            </Dropdown>
+                          )
+                        }}
+                      >
+                        <CardTitle>
+                          <Label icon={<PficonTemplateIcon />} color="green">
+                            Draft
+                          </Label>
+                          {draft.title ? ` ${draft.title}` : ` Untitled ${index + 1}`}
+                        </CardTitle>
+                      </CardHeader>
+                      <CardBody>
+                        <Flex justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+                          <FlexItem>Branch name: {draft.branchName}</FlexItem>
+                          <FlexItem>State: Draft</FlexItem>
+                          <FlexItem>Last updated: {draft.lastUpdated}</FlexItem>
+                          <FlexItem>
+                            {draft.isKnowledgeDraft ? (
+                              <Label key="knowledge" color="blue" style={{ marginRight: '5px' }}>
```

Comment:
why do we need the 5px right margin?

Reply:
without the margin, there is no space between labels. Anything better we can do here?

Reply:
Yeah, if we add a `gap="gapSm"` to the containing `Flex` component you can remove the margin.